doc/classes/Color.xml

```diff
@@ -19,7 +19,7 @@
 			<return type="Color" />
 			<description>
 				Constructs a default [Color] from opaque black. This is the same as [constant BLACK].
-				[b]Note:[/b] in C#, constructs an empty color with all of its components set to [code]0.0[/code] (transparent black).
+				[b]Note:[/b] In C#, constructs a transparent black color with all of its components set to [code]0.0[/code]. For an opaque black, use [constant BLACK] instead.
```

Comment:
_"For opaque black, use [constant BLACK] instead."_ feels redundant. The prior description already implies this.
```suggestion
				[b]Note:[/b] In C#, this constructs a transparent black color, with all of its components set to [code]0.0[/code].
```
or perhaps for consistency with the other notes:
```suggestion
				[b]Note:[/b] In C#, this constructs a [Color] with all of its components set to [code]0.0[/code] (transparent black).
```